climate-monitoring-client/src/main/java/climatemonitoring/CenterCreation.java

```diff
@@ -1,18 +1,96 @@
 package climatemonitoring;
 
+import climatemonitoring.core.Center;
+import climatemonitoring.core.ConnectionLostException;
+import climatemonitoring.core.DatabaseRequestException;
 import climatemonitoring.core.ViewState;
+import climatemonitoring.core.headless.Console;
 
 class CenterCreation extends ViewState {
 
 	@Override
 	public void onHeadlessRender(String args) {
 
-		throw new UnsupportedOperationException("Unimplemented method 'onHeadlessRender'");
+		String errorMsg = null;
+
+		do {
+
+			m_centerID = Console.read("Center ID > ");
+			errorMsg = Check.centerID(m_centerID);
+
+			if (errorMsg != null)
+				Console.write(errorMsg);
+
+		} while (errorMsg != null);
+
+		do {
+
+			m_city = Integer.parseInt(Console.read("City > "));
+			m_street = Console.read("Street > ");
+			m_houseNumber = Integer.parseInt(Console.read("House number > "));
+			errorMsg = Check.address(m_city, m_street, m_houseNumber);
+
+			if (errorMsg != null)
+				Console.write(errorMsg);
+
+		} while (errorMsg != null);
```

Comment:
You should check for a `NumberFormatException` at line 28 and 30

Reply:
Fixed